### PR TITLE
Sand and leather

### PR DESCRIPTION
**Pull request by pervertedPistachio**
#### Summary
Bugfixes "Sand extraction and free leather patches"

1. Changed Tanned hide recipe
2. added ability to extract sand from sand mounds

#### Purpose of change

- There are exploits that allow players to create leather patches out of thin air by creating and deconstructing leather hide
- player can extract sand from sand mounds similar to clay mounds

Fixes: #59068 & #58767 

#### Describe the solution

- changed recipe for tanned hide to match deconstruction result
- allow player to extract sand from mounds
- 
#### Describe alternatives you've considered

N/A

#### Testing

Crafted and deconstructed Tanned hide

extracted sand from sand mound

#### Additional context

Tanned hide
Mound2
Mound

### CODE REVIEW

**Review by Maleclypse — APPROVED** [review]

**Comment by Devengel** [comment]
the sand extraction fix is already in #59095

**Comment by Night-Pryanik** [comment]
For the future, please don't combine several unrelated PRs into one. For this one I recommend you to leave only leather-related changes since sand-related stuff is already covered by #59095.

File: data/json/construction.json [hunk 1]
```diff
@@ -4016,6 +4017,19 @@
     "pre_terrain": "t_sand",
     "post_special": "done_extract_maybe_revert_to_dirt"
   },
+  {
+    "type": "construction",
+    "id": "constr_extract_sand_mound",
+    "group": "extract_sand",
+  	"//": "Step 2 : extract sand",
```
Comment (BrettDong) [inline data/json/construction.json:4024]: Two spaces indentation
```suggestion
  	 "//": "Step 2 : extract sand",
```

File: data/json/construction.json [hunk 2]
```diff
@@ -4008,6 +4008,7 @@
     "type": "construction",
     "id": "constr_extract_sand",
     "group": "extract_sand",
+    "//": "Step 1 : extract sand",
```
Comment (Maleclypse) [inline data/json/construction.json:4011]:
```suggestion
```

File: data/json/construction.json [hunk 3]
```diff
@@ -4016,6 +4017,19 @@
     "pre_terrain": "t_sand",
     "post_special": "done_extract_maybe_revert_to_dirt"
   },
+  {
```
Comment (Maleclypse) [inline data/json/construction.json:4020]:
```suggestion
```

File: data/json/construction.json [hunk 4]
```diff
@@ -4016,6 +4017,19 @@
...
+    "type": "construction",
```
Comment (Maleclypse) [inline data/json/construction.json:4021]:
```suggestion
```

File: data/json/construction.json [hunk 5]
```diff
@@ -4016,6 +4017,19 @@
...
+    "id": "constr_extract_sand_mound",
```
Comment (Maleclypse) [inline data/json/construction.json:4022]:
```suggestion
```

File: data/json/construction.json [hunk 6]
```diff
@@ -4016,6 +4017,19 @@
...
+    "group": "extract_sand",
```
Comment (Maleclypse) [inline data/json/construction.json:4023]:
```suggestion
```

File: data/json/construction.json [hunk 7]
```diff
@@ -4016,6 +4017,19 @@
...
+  	 "//": "Step 2 : extract sand",
```
Comment (Maleclypse) [inline data/json/construction.json:4024]:
```suggestion
```

File: data/json/construction.json [hunk 8]
```diff
@@ -4016,6 +4017,19 @@
...
+    "category": "OTHER",
```
Comment (Maleclypse) [inline data/json/construction.json:4025]:
```suggestion
```

File: data/json/construction.json [hunk 9]
```diff
@@ -4016,6 +4017,19 @@
...
+    "required_skills": [ [ "survival", 0 ] ],
```
Comment (Maleclypse) [inline data/json/construction.json:4026]:
```suggestion
```

File: data/json/construction.json [hunk 10]
```diff
@@ -4016,6 +4017,19 @@
...
+    "time": "30 m",
```
Comment (Maleclypse) [inline data/json/construction.json:4027]:
```suggestion
```

File: data/json/construction.json [hunk 11]
```diff
@@ -4016,6 +4017,19 @@
...
+    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
```
Comment (Maleclypse) [inline data/json/construction.json:4028]:
```suggestion
```

File: data/json/construction.json [hunk 12]
```diff
@@ -4016,6 +4017,19 @@
...
+    "byproducts": [ { "item": "material_sand", "charges": [ 300, 600 ] } ],
```
Comment (Maleclypse) [inline data/json/construction.json:4029]:
```suggestion
```

File: data/json/construction.json [hunk 13]
```diff
@@ -4016,6 +4017,19 @@
...
+    "pre_terrain": "t_sandmound",
```
Comment (Maleclypse) [inline data/json/construction.json:4030]:
```suggestion
```

File: data/json/construction.json [hunk 14]
```diff
@@ -4016,6 +4017,19 @@
...
+    "post_special": "done_extract_maybe_revert_to_dirt"
```
Comment (Maleclypse) [inline data/json/construction.json:4031]:
```suggestion
```

File: data/json/construction.json [hunk 15]
```diff
@@ -4016,6 +4017,19 @@
...
+  },
```
Comment (Maleclypse) [inline data/json/construction.json:4032]:
```suggestion
```

**Pull request merged**